src/implementations/twig/components/link/link.story.js

```diff
@@ -26,86 +25,110 @@ const withParagraph = (story) => {
   return typeof demo === 'string' ? storyAsString(demo) : storyAsNode(demo);
 };
 
-const iconsList = {};
-iconsList.none = '';
-
-uiIcons.forEach((icon) => {
-  iconsList[icon] = icon;
-});
+const iconsList = uiIcons.reduce((a, b) => {
+  a[b] = b;
+  return a;
+}, {});
+
+const getArgTypes = (data) => {
+  return {
+    label: {
+      name: 'label',
+      type: { name: 'string', required: true },
+      defaultValue: data.link.label,
+      description: 'The main label of the button',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '' },
+        category: 'Content',
+      },
+      control: {
+        type: 'text',
+      },
+    },
+    icon_position: {
```

Comment:
I think it would be good to align with the icon controls, see button controls. I think the control for the position should be last.  I'm also using inline-select type for this field control. Also in the buttons the position of the icon by default is "after", which does not seem to be the case with links, we can maybe update the default value for links?

I'm using "icon name" and here you use "icon (sample)" for the icon control label. 

I think we have to align either the links or the buttons and use the same approach for the icons controls?

Reply:
I reproduced what we have in ECL v2, which it was what we agreed to do...
There is no default for the icon_position in the link component, i see in the "docs" as such but the documentation is not aligned with the template.
I'll align this pr with what has been merged already, then, but i'm not sure whether to fix the template by setting a default value for the icon_position or fixing the documentation, instead.
I'll do the second, still we can then decide to apply a default.